packages/larva/.eslintignore

```diff
@@ -0,0 +1,6 @@
+lib/**/*.js
```

Comment:
Note: these will be lint-fix'd and committed in a separate PR to keep this one smaller.